packages/framework/src/Form/OrderItemsType.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Form;
+
+use Shopsys\FrameworkBundle\Form\Admin\Order\OrderItemFormType;
+use Shopsys\FrameworkBundle\Form\Admin\Order\OrderPaymentFormType;
+use Shopsys\FrameworkBundle\Form\Admin\Order\OrderTransportFormType;
+use Shopsys\FrameworkBundle\Model\Order\Item\OrderItemPriceCalculation;
+use Shopsys\FrameworkBundle\Model\Order\Order;
+use Shopsys\FrameworkBundle\Model\Payment\PaymentFacade;
+use Shopsys\FrameworkBundle\Model\Transport\TransportFacade;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CollectionType;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class OrderItemsType extends AbstractType
```

Comment:
This new form element should probably be described in [the docs](https://github.com/shopsys/shopsys/blob/v7.0.0-beta2/docs/wip_glassbox/form-extension.md#what-can-we-use-for-creating-our-own-forms).

Reply:
And I'm thinking if the order items should be extensible as well. Probably not, because of [the problems with extending the OrderItem](https://github.com/shopsys/shopsys/blob/v7.0.0-beta2/docs/wip_glassbox/entity-extension.md#orderitem-and-true-mapped-inheritance), right? If so, we might want to update US-3727 in our internal backlog to mention the form extensibility.

Reply:
After discussion we will leave the OrderItem forms inextensible, as mentioned in https://github.com/shopsys/shopsys/pull/576#discussion_r232175285

Reply:
done